internal/rego/rego.go

```diff
@@ -181,18 +181,21 @@ func (r Rego) AnnotationParameters() map[string]apiextensionsv1.JSONSchemaProps
 	return r.annoParameters
 }
 
-func (r Rego) GetAnnotation(name string) (any, bool) {
+func (r Rego) GetAnnotation(name string) (any, error) {
```

Comment:
Had to change the signature here due to https://go.dev/src/text/template/funcs.go?s=14790:14826